src/extension.ts

```diff
@@ -7,7 +7,9 @@ import {
   CommaPosition,
   LogicalOperatorNewline,
   FormatOptionsWithLanguage,
+  FormatOptions
 } from 'sql-formatter';
+import { ParamTypes } from 'sql-formatter/lib/src/lexer/TokenizerOptions';
```

Comment:
Instead of reaching deep inside the source tree (which might break with even the next patch release) we can simply access `ParamTypes` through:

```ts
type ParamTypes = FormatOptions["paramTypes"];
// or
type ParamTypes = NonNullable<FormatOptions["paramTypes"]>;
```

Reply:
Though this type really should be exported directly by `sql-formatter`.

I added an issue: https://github.com/sql-formatter-org/sql-formatter/issues/577

Reply:
@nene Thanks for the review and so sorry for the delay to respond. I applied your feedback and want to merge the change but are there any process to request merge?

Reply:
Well, I could merge this... but it's that only @inferrinizzard can actually release it. And he's rarely around.

Reply:
Ah I see, then could you please merge this? Meanwhile I'll use private build version extension until the change is officially released.

Reply:
Sure. You're welcome.